.gitignore

```diff
@@ -4,3 +4,4 @@ tests/config.php
 vendor
 .phpunit.result.cache
 .phplint-cache
+.idea
```

Comment:
this doesn't belong here

Reply:
@GrahamCampbell my idea was mostly that if people fork and want to contribute for a first time to not do any silly mistakes and add their phpstorm/idea config files to the project. If it's not wanted i'll remove it

Reply:
I don't have a strong preference, and it seems a bit too categorical to say it can't be here. I suppose we need to set the line between local dev setup (which should probably be in `~/.gitignore`). Let's remove it.